aries_cloudagent/messaging/socket.py

```diff
@@ -116,6 +116,12 @@ def select_outgoing(self, message: OutboundMessage) -> bool:
                 and message.reply_to_verkey in self.reply_verkeys
             ):
                 return True
+            if (
+                mode == self.REPLY_MODE_ALL
+                and any(True for k in message.target.recipient_keys
```

Comment:
I'm not sure that `message.target` is guaranteed to be set.

Reply:
fixed with additional checks